src/Integration.UnitTests/Suppression/SonarQubeIssuesProviderTests.cs

```diff
@@ -211,9 +211,9 @@ public void GetIssues_NoIssuesOnServer_ReturnsEmptyList()
         public void GetSuppressedIssues_WhenProjectHasModulesAndIssueIsModuleLevelFound_ReturnsExpectedIssue()
         {
             // Arrange
-            var sonarQubeIssue1 = new SonarQubeIssue(null, null, "message", "sqkey:sqkey:projectId2", "S1", true, SonarQubeIssueSeverity.Blocker, null, flows: null);
```

Comment:
There are only three important fields used by these tests, so I created a helper function that also handles setting the new fields appropriately.